### ISSUE TITLE
[Bug][A11Y][Input Tag] The whole component should have the role button

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#481]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

iOS 26

### App version

0.6.0

### Describe the issue

The role "button" is set to the cross only.
The whole component should have the role.

### Expected Behavior

The accessible name should be constructed with the displayed "Label" + "remove tag"

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: DesignToolbox (ALPHA) (693)
Version: 2.0.0
Build type: ALPHA
Build details (GitHub): 481
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr ) to get access.

Display name: Design Toolbox Flutter (693)
Version: 2.0.0
Build type: ALPHA
Build details (GitHub): 481
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
tested OK on Android
